flexmeasures/ui/static/js/flexmeasures.js

```diff
@@ -311,13 +311,29 @@ function submit_sensor_type() {
     $("#sensor_type-form").attr("action", empty_location).submit();
 }
 
-/** Tooltips: Register custom formatter for quantities incl. units
-              Usage:
-                  {
-                      'format': [<d3-format>, <sensor unit>],
-                      'formatType': 'quantityWithUnitFormat'
-                  }
-*/
+/** Tooltips: Register custom formatters
+ *
+ *          - Quantities incl. units
+ *            Usage:
+ *                 {
+ *                     'format': [<d3-format>, <sensor unit>],
+ *                     'formatType': 'quantityWithUnitFormat'
+ *                 }
+ *          - Timedeltas measured in human-readable quantities (usually not milliseconds)
+ *            Usage:
+ *                 {
+ *                     'format': [<d3-format>, <breakpoint>],
+ *                     'formatType': 'timedeltaFormat'
+ *                 }
+ */
 vega.expressionFunction('quantityWithUnitFormat', function(datum, params) {
     return d3.format(params[0])(datum) + " " + params[1];
 });
+vega.expressionFunction('timedeltaFormat', function(timedelta, params) {
```

Comment:
Can we document here what I had to find out by myself?

params[0] is a D3 format identifier (.e.g "d" for days)
params[1] is a multiplier which decides from how many X on we format as Y (e.g. <=3 days means we format as "hours")

Reply:
Actually, d stands for decimal notation, rounded to integer. It's a [d3-format](https://github.com/d3/d3-format ) rather than a [d3-time-format](https://github.com/d3/d3-time-format). I'm expanding the inline documentation.